scripts/choco-install.ps1

```diff
@@ -0,0 +1,25 @@
+# Add the cached packages with source priority 1 (Chocolatey community is 0)
+choco source add --name="nodejs" --source=".\tmp\chocolatey\nodejs.install" --priority=1
+choco source add --name="python" --source=".\tmp\chocolatey\python3" --priority=1
```

Comment:
The answer and docs seem to mean that this has to be done repeated for each individual package. But I wonder if it would work if you just did:

```ps
choco source add --source=".\tmp\chocolatey"
```

Reply:
Additionally, rather than using `.\tmp` which is dependent on script execution environment, instead use:

```
"${PSScriptRoot}\..\tmp\chocolatey"
```

The `$PSScriptRoot` refers to the directory in which the script is located. So we go up one directory and down to `tmp`.

Reply:
Try to make sure you have created the `${PSScriptRoot}\..\tmp\chocolatey` first with a `New-Item`. Make it silent as well.

Reply:
Specifying the parent directory as a source works, so leaving it as that.